README.md

```diff
@@ -1,13 +1,13 @@
 <!-- Image sourced from https://blog.1password.com/introducing-secrets-automation/ -->
 <img alt="" role="img" src="https://blog.1password.com/posts/2021/secrets-automation-launch/header.svg"/>
 
-<header style="text-align: center;">
-	<h1 style="margin-top: 20px; border-bottom: none;">1Password Connect SDK JS</h1>
+<div align="center">
```

Comment:
Isn't `align` an obsolete attribute? 🤔

Reply:
[It is](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p#attributes) indeed! But apparently GitHub still respects it and renders it while ignoring the modern equivalent of `style="text-align: center;"`.

Reply:
Well that's lovely. 🤦🏻 
Guess we go this way then.